packages/nx-semantic-release/src/tests/setup-test-repo.ts

```diff
@@ -288,6 +288,8 @@ export async function setupTestRepo(
 
   await bootstrapTestProjectsAndLibs();
 
+  await runCommandsInTestProj(['npm install -D ajv']);
```

Comment:
For some reason ajv wasn't installed correctly. And plain `npm install` didn't work for me at least locally. This solves the issues. Hopefully it can be removed in future. Maybe after Nx has been upgraded to latest